src/main/resources/db/migration/V112__edit_ethnicity_code_descriptions.sql

```diff
@@ -0,0 +1,45 @@
+BEGIN;
+-------------------------------------------------------
+UPDATE ethnicity_codes
+SET description = 'Asian/Asian British : Chinese'
+WHERE id = 4;
```

Comment:
Are these IDs going to be the same across all environments?
Could we use the code instead?